backend/rates/history_test.go

```diff
@@ -42,7 +42,10 @@ func TestPriceAt(t *testing.T) {
 		{7.25, time.Date(2020, 9, 3, 18, 0, 0, 0, time.UTC)},
 	}
 	for _, test := range tt {
-		assert.InDelta(t, test.wantValue, updater.HistoricalPriceAt("btc", "USD", test.at), 0, "at = %s", test.at)
+		//nolint:testifylint
```

Comment:
There are 4 other instances of InDelta that you could revert.

Reply:
A, ok, i thought you mean it just for that one case. I'll revert it and exclude the rule globally.